torch/_ops.py

```diff
@@ -352,6 +352,7 @@ def handler(*args, **kwargs):
             return handler
 
         final_key = resolve_key(self, key)
+        # print(self, key, final_key)
```

Comment:
not - debug comment

Reply:
btw this is a SUPER useful debugging tool if you have a minimal example, it's about as useful as dispatch trace but you don't have to compile with DEBUG to get it

Reply:
haha 100% agreed.